deno.json

```diff
@@ -5,5 +5,11 @@
     "publish": "deno task build-npm && cd npm/ && npm publish",
     "build-npm": "deno run -A scripts/build-npm.ts",
     "docs": "deno doc --html --name='domain-functions' ./mod.ts"
+  },
+  "lint": {
+    "include": ["src/"],
+    "rules": {
+      "exclude": ["no-explicit-any", "ban-types"]
```

Comment:
`ban-types` is the lint rule that will forbid `{}` (since one usually want to use a `Record`). This type we could ban in the future but for now I rather reduce the linting noise.

And explicit `any` is very useful in this code-base, so it surprises me we haven't done this sooner :laughing:

Reply:
I don't like that linting rule because I like to use `{}` as it is equivalent to something like `Omit<any, undefined | null>`